pkgs/by-name/kl/klog-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "klog-rs";
+  version = "0.0.3";
+
+  src = fetchFromGitHub {
+    owner = "tobifroe";
+    repo = "klog";
+    rev = version;
+    hash = "sha256-HEGxg277483ZZpXKFIBTAITKie/iBfbz9KmtqzlGC3E=";
+  };
+  cargoHash = "sha256-mOFzlq5AEks9vwjk2FxTGLyw6RkI44kml2t8r1UsdOk=";
+  checkFlags = [
+    # this integration test depents on a running kubernetes cluster
+    "--skip=k8s::tests::test_get_pod_list"
+  ];
+
+  meta = {
+    description = "klog is a tool that allows you to tail logs of multiple Kubernetes pods simultaneously.";
```

```suggestion
    description = "Tool to tail logs of multiple Kubernetes pods simultaneously";
```
See the [Meta Attributes chapter](https://nixos.org/manual/nixpkgs/unstable/#var-meta-description) of the Nixpkgs Manual.

---

Adapted the description, thank you!